data/json/items/classes/comestible.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": "food",
+    "type": "COMESTIBLE",
+    "name": "edible comestible",
+    "comestible_type": "FOOD",
+    "volume": 1,
+    "symbol": "%"
+  },
+  {
+    "id": "fruit",
+    "copy-from": "food",
+    "type": "COMESTIBLE",
+    "name": "base fruit",
+    "material": "fruit",
+    "color": "red",
```

Comment:
Same with color here, it's not always red.  Though in this case I believe you've arbitrarily changed the color of many fruits to match your template.